Description: Django 4.x: django.conf.urls.url is removed
 Note: upstream already has a patch for it, though mixed with other
 stuff, so not using it directly.
Bug-Debian: https://bugs.debian.org/1013603
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2022-06-30

--- python-enmerkar-0.7.1.orig/tests/testproject/urls.py
+++ python-enmerkar-0.7.1/tests/testproject/urls.py
@@ -1,6 +1,6 @@
 import time
 
-from django.conf.urls import url
+from django.urls import re_path
 from django.shortcuts import render
 from django.utils.timezone import now
 
@@ -14,5 +14,5 @@ def test_view(request):
 
 
 urlpatterns = [
-    url('^$', test_view),
+    re_path('^$', test_view),
 ]
